lib/gutdata/core/logging.rb

```diff
@@ -1,43 +1,50 @@
 # encoding: UTF-8
 #
-# Copyright (c) 2010-2015 GoodData Corporation. All rights reserved.
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+require 'rest-client'
+
 require_relative 'nil_logger'
 
 module GutData
+  DEFAULT_LOG_LEVEL = Logger::INFO
+  DEFAULT_LOG_OUTPUT = STDOUT
+  DEFAULT_LOGGER_CLASS = Logger
+
+  DEFAULT_RESTLOG_LEVEL = Logger::DEBUG
+  DEFAULT_RESTLOG_OUTPUT = STDOUT
+  DEFAULT_RESTLOGGER_CLASS = Logger
+
   class << self
-    attr_writer :logger
+    attr_accessor :logger, :rest_logger
     attr_writer :stats
 
     # Turn logging on
     #
     # ### Example
     #
-    #     GutData.logging_on
+    #     # Turn of default logging
```

Comment:
```suggestion
    #     # Turn on default logging
```

Reply:
I'm trying to keep the true intent behind the original GoodData code here of course /s